tools/test

```diff
@@ -46,14 +47,14 @@ while (( $# )); do
         --diff) shift; files=diff:"$1"; shift;;
         --full) files=full; shift;;
         --fix) fix=1; shift;;
-        android|flow|lint|jest|prettier|spell)
+        android|flow|lint|jest|prettier|spell|functional)
             suites+=("$1"); shift;;
         *) usage;;
     esac
 done
 
 if [ -z "$suites" ]; then
-    suites=(android flow lint jest prettier)
+    suites=(android flow lint jest prettier functional)
```

Comment:
I think we'll want to leave it out of these defaults (for when the test script is run without arguments), so you say `tools/test functional` when you want to run it. Then we can just say that in the Travis config to make it run in CI.

That takes it out of the hot loop in normal development (edit, run tests, edit, run tests, etc.) -- we try to keep that very fast, like a few seconds.

Reply:
Sure makes complete sense.  I have now removed `functional` test suite from default test suite.